### PR TITLE
IP 2482/v7.3 gms additions

### PR DESCRIPTION
**Pull request by cjohnson12**
Updated:
Morphology and Topography for TumourSample
Collection field in NGIS Referral for Research Cohorts
GMS Consent enums updated

### CODE REVIEW

File: schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl [hunk 1]
```diff
@@ -419,40 +417,54 @@ protocol CommonParticipant {
         union {null, string} clinicalIndicationVersionNumber;
     }
 
+    record Identifier{
```
Comment (priesgo) [inline schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl]: This name causes a collision with another record called `Identifier`.
Also, we already have something for this in the models called `Property` or alternatively a more complex one `StandardPhenotype`

Comment (Antonior26) [reply schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl]: Yep Agree, I would do the changes tomorrow, first thing in the morning @priesgo

File: schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl [hunk 2]
```diff
@@ -280,9 +280,7 @@ protocol CommonParticipant {
     clinicalEthnicities supersedes Ancestries in GMS
     */
 
-    enum GenericConsent {`true`, `false`, not_discussed}
-
-    enum PrimaryConsent {`true`, `false`, best_interest}
+    enum GenericConsent {YES, NO, NOT_APPLICABLE, UNDEFINED}
```
Comment (Antonior26) [inline schemas/IDLs/org.gel.models.participant.avro/1.2.0/CommonParticipant.avdl]: This should be lower case

**Comment by Antonior26** [comment]
@cjohnson12 you have done changes (GMS consent that were already done and approved) - therefore I am closing this PR and starting over.

**Pull request closed without merging**